### ISSUE TITLE
Add option to change x plot Axis while maintaining time conductor query

### ISSUE THREAD

**Issue by mudinthewater** [nasa/openmct#2380]
Would work the same way that the Y-axis drop down does in overlay plots. 

Have a use-case where a user would like to query by ERT but plot by SCET range returned. Mostly good for testbed work where scets are pretty useless (they overlap) except in the context of ERT. 

Not sure if this is doable in the current version of plotting where data outside the query range is ignored. In the very old version of OpenMCT from a few years ago, it would plot everything you sent it, which I used as a cheat to accomplish this (I'm terrible).

**Comment by charlesh88** [comment]
So we'd do something like this?
1. Time Conductor set to Fixed ERT with start and end in that format, MCWS returns a bunch 'o data.
1. User changes plot X axis from ERT to SCET.
1. We determine the earliest and latest SCETs in the current data bounds, then re-query MCWS for that plot alone.
1. Plot displays with well-ordered data based on returned SCET values.
1. If the user diddles (this is a technical term) the Time Conductor settings, the plot repeats step 3 and 4 automatically.

Is this doable with real-time data?

**Comment by mudinthewater** [comment]
nooooooooooooooo

**Comment by mudinthewater** [comment]
That would be bad

**Comment by mudinthewater** [comment]
Basic idea is this - you have a dataset that you want to see in SCET. Let's say this represents a maneuver. 

Your dataset took 5s to process on tuesday at 1:05 pm. So you make a query in ERT from 1:04 to 1:05. However, in SCET, the data was actually 4 hours of data, _on a completely different day_, say wednesday between 1pm and 4pm. You have processed this dataset with 10 different DN/EU conversions about 50x. You don't want to see this data in ERT, where your 4 hours will be squashed into 5s of plot. If you re-query in SCET like you say above, you will get the 49 other times you didn't want. 

Basically you'd need to stop dropping data that isn't within the bounds of the original query, then plot with the x-axis that the user selected. 

For what it's worth, this is what session filtering is sort of meant to accomplish.

**Comment by akhenry** [comment]
@mudinthewater Thanks for the details
@charlesh88 If I'm understanding what @mudinthewater is asking for, I *think* we can achieve it with what we discussed yesterday afternoon, which doesn't require re-querying. The ERT query gives us all of the data we need, we just need to plot it by SCET instead of ERT. No re-query necessary. We can do this by simply adding an x-axis selector. Actually we'd be restoring something that used to be there (not actually sure what happened to it? Maybe it was removed with the re-implementation of the time conductor to force the user to use the system-wide canonical time system, without considering this particular use case?).

**Comment by unlikelyzero** [comment]
Will verify in next testathon

**Comment by unlikelyzero** [comment]
@kobe1104 and @davetsay to retest in VISTA next testathon

**Comment by akhenry** [comment]
@davetsay  Can we mark this verified yet?
